CHANGELOG.md

```diff
@@ -3563,6 +3566,11 @@ Additions to existing modules
   poset          : Set a → Poset _ _ _
   ```
 
+* Added new proof in `Relation.Nullary.Reflects`:
+  ```agda
+  T-reflects : Reflects (T b) b
+  ```
```

Comment:
So, I chose the name `T⁺` in #2055 on the model of the style-guide for "pre- and post-conditions". I don't disagree with the new name per se, but then what would you call `T⁻` defined below? In any case, I don't think these names should ever be exported  outside `Relation.Nullary.Decidable.Core`...?

Reply:
I chose this name because it was consistent with the other properties. I don't think this falls into `pre- and post- conditions` convention in the style guide. 

> When defining a new relation P over a datatype X in a Data.X.Relation module, it is often common to define how to introduce and eliminate that relation with respect to various functions.

Firstly, we're not in a `Data.X.Relation` module. Secondly, I guess you could argue that `P` is `Reflects` but then what is the datatype `X` and what is the function that eliminates or introduces it?

I'll have a think about a suitable name.

Reply:
I think that's a fair response (modulo my regarding `Reflects` as the datatype, or else as a `Set`-valued view of `Bool`, but in any case, as an inductive family; and then the 'function' is... `T` itself; but I admit those are the possibly-unconventional spectacles through which I look at the world), and given the rest of `Reflects`, the choice of name here seems fine... up to what its dual/inverse should be called. But I do think the principle that, just as with introduction/elimination, these things 'come in pairs', should be adhered to wherever possible. The `T⁺`/`T⁻` scheme is then a convenient, and minimum-ink, reification of that principle. But this probably belongs in a separate `style-guide` collection of issues... ;-)